doc/how-to/replication/repl_bootstrap_auto.rst

```diff
@@ -0,0 +1,437 @@
+           • Connecting to the instance...
+           • Connected to auto_leader:instance001
+
+2.  Check the instance state in regard to :ref:`leader election <repl_leader_elect_process>` using :ref:`box.info.election <box_info_election>`.
+    The output below shows that ``instance001`` is a follower while ``instance002`` is a replica set leader.
+
+    .. code-block:: console
+
+        auto_leader:instance001> box.info.election
+        ---
+        - leader_idle: 0.77491499999815
+          leader_name: instance002
+          state: follower
+          vote: 0
+          term: 2
+          leader: 1
+        ...
+
+3.  Check that ``instance001`` is in read-only mode using ``box.info.ro``:
+
+    .. code-block:: console
+
+        auto_leader:instance001> box.info.ro
+        ---
+        - true
+        ...
+
+4.  Execute ``box.info.replication`` to check a replica set status.
+    Make sure that ``upstream.status`` and ``downstream.status`` are ``follow`` for ``instance002`` and ``instance003``.
+
+    .. code-block:: console
+
+        auto_leader:instance001> box.info.replication
+        ---
+        - 1:
+            id: 1
+            uuid: 4cfa6e3c-625e-b027-00a7-29b2f2182f23
+            lsn: 9
+            upstream:
+              status: follow
+              idle: 0.8257709999998
+              peer: replicator@127.0.0.1:3302
+              lag: 0.00012326240539551
+            name: instance002
+            downstream:
+              status: follow
+              idle: 0.81174199999805
+              vclock: {1: 9}
+              lag: 0
+          2:
+            id: 2
+            uuid: 9bb111c2-3ff5-36a7-00f4-2b9a573ea660
+            lsn: 0
+            name: instance001
+          3:
+            id: 3
+            uuid: 9a3a1b9b-8a18-baf6-00b3-a6e5e11fd8b6
+            lsn: 0
+            upstream:
+              status: follow
+              idle: 0.83125499999733
+              peer: replicator@127.0.0.1:3303
+              lag: 0.00010204315185547
+            name: instance003
+            downstream:
+              status: follow
+              idle: 0.83213399999659
+              vclock: {1: 9}
+              lag: 0
+        ...
+
+
+.. _replication-automated-failover-add-data:
+
+Adding data
+~~~~~~~~~~~
+
+To check that replicas (``instance001`` and ``instance003``) get all updates from the master(``instance002``), follow the steps below:
+
+1.  Connect to ``instance002`` using ``tt connect``:
+
+    .. code-block:: console
+
+        $ tt connect auto_leader:instance002
+           • Connecting to the instance...
+           • Connected to auto_leader:instance002
+
+2.  Create a space and add data as described in :ref:`CRUD operation examples <box_space_examples>`.
+
+3.  Use the ``select`` operation on ``instance001`` and ``instance003`` to make sure data is replicated.
+
+4.  Check that the 1-st component of :ref:`box.info.vclock <box_introspection-box_info>` values are the same on all instances:
```

Comment:
`1-st` looks a bit weird.
I get that it's not the `first` but `corresponding to key 1`. Maybe call it "``1`` component" ("1" digit in code style)?

Reply:
Agree, took this from the old changelog. Will fix.